backends/google/sheets/google-sheets.js

```diff
@@ -318,7 +316,7 @@ export default class GoogleSheets extends Google {
 	 * @returns The range reference in one of the supported formats: 'Sheet title'!Range, 'Sheet title', or Range.
 	 */
 	static #getRangeReference ({sheet, range} = file) {
-		return `${sheet ? `'${sheet}'` : ""}${range ? (sheet ? `!${range}` : range) : ""}`
+		return `${sheet ? `'${sheet}'` : ""}${range ? (sheet ? `!${range}` : range) : ""}`;
```

Comment:
We need an ESLint file in this repo…

Reply:
This is how I've caught these issues—by setting ESLint up. :) So, yes, having such a file would be nice. It will also force the code style we follow.

And I think we should gitignore node_modules too.